scripts/make_giantsteps_key.py

```diff
@@ -57,5 +57,7 @@ def main(args):
 
 if __name__ == '__main__':
     PARSER = argparse.ArgumentParser(description='Make giantsteps_key index file.')
-    PARSER.add_argument('giantsteps_key_data_path', type=str, help='Path to giantsteps_key data folder.')
+    PARSER.add_argument(
+        'giantsteps_key_data_path', type=str, help='Path to giantsteps_key data folder.'
+    )
```

Comment:
This is being edited by black though normally it would not be part of the PR. Maybe try in future PRs to run `black -S` and will skip strings. We will fix this in the Docs audit later on #245

Reply:
you can do this to exclude it, provided that origin/master is up to date
```
git checkout origin/master -- scripts/giantsteps_key.py
git commit
git push
```